app/views/notes/show.html.erb

```diff
@@ -86,6 +86,10 @@
     </table>
   <% end %>
 
+  <% if current_user && ( current_user.role == "admin" || current_user.role == "moderator") && !(@node.has_tag('locked')) %>
+    <a href="javascript:window.location.reload(true)" onClick="addTag('locked');">Lock Note</a>
```

Comment:
I think this may be an issue in Firefox, but I could be wrong. I noted this in https://github.com/publiclab/plots2/pull/9726 too!

Reply:
@jywarren I use Firefox and it works fine!

Reply:
@Manasa2850 I just tested the `Lock Note` option on the stable site, and on the first click, the reload does not add the tag, I have to click it again or manually reload the page. I am using a chrome browser. 


https://user-images.githubusercontent.com/7622875/121544523-7836a000-ca12-11eb-85cb-579019d695d6.mov

Reply:
@RuthNjeri I'm not really sure why that's happening on the stable site. I tried it out on localhost both on Chrome as well as Firefox and the tag gets added on reload. There's no need to refresh the page. 
![ezgif com-gif-maker](https://user-images.githubusercontent.com/43354059/121583761-2e54b680-ca20-11eb-8088-623eaead2ce1.gif)

I'll take a deeper look at this to find out why that's happening on the stable site.

Reply:
I also think you need to be an admin to test this out on the stable site, I don't know how to make someone an admin on the stable site, but if you share your username with Jeff, he will be able to do that.

Reply:
My username on the stable site is Manasa2850.

Reply:
@RuthNjeri I tried it out on stable and this is what I observer
1. On Firefox - it works perfectly fine. We don't need to refresh the page for the locked tag to get added.
![stable-firefox](https://user-images.githubusercontent.com/43354059/121686787-3fb0c800-cadf-11eb-8bd5-01c20254feb8.gif)

2. On Chrome - sometimes it works fine, without requiring a refresh. But few other times, on clicking the `Lock Note` button, the note gets locked (i.e the lock favicon appears and the Lock Note button goes away) but the 'locked' tag card doesn't appear in the sidebar. We need to refresh the page in order to see it on the sidebar.
![stable-chrome](https://user-images.githubusercontent.com/43354059/121687067-8999ae00-cadf-11eb-8448-415d5c73244b.gif)

I will try to find out why this is happening just on Chrome. Thanks!

Reply:
Thanks for the update @Manasa2850